Dockerfile

```diff
@@ -9,8 +8,9 @@ COPY . /RackHD/on-tasks/
 
 RUN cd /RackHD/on-tasks \
   && mkdir -p /RackHD/on-tasks/node_modules \
+  && npm install --ignore-scripts --production \
+  && rm -r /RackHD/on-tasks/node_modules/on-core \
```

Comment:
Does `di` need to be removed if `on-core` needs to be removed, both of which are made a link later?